recipes/rgb

```diff
@@ -0,0 +1,4 @@
+(rgb
+ :fetcher gitlab
+ :repo "cwpitts/rgb.el"
+ :files ("rgb.el"))
```

Comment:
I'm not saying you must, but having `:defaults` in the recipe gives you room to grow and is slightly preferable:
```suggestion
(rgb
 :fetcher gitlab
 :repo "cwpitts/rgb.el"
 :files (:defaults
         (:exclude "el-mock.el"
                   "ert-expectations.el"
                   "test-rgb.el")))
```
And if you renamed test-rgb.el to rgb-test.el, it will automatically be excluded by `:defaults`, and in that case you could write:
```suggestion
(rgb
 :fetcher gitlab
 :repo "cwpitts/rgb.el"
 :files (:defaults
         (:exclude "el-mock.el" "ert-expectations.el"))
```

Reply:
Suggestions applied!